draft-ietf-oauth-status-list.md

```diff
@@ -151,6 +151,9 @@ Status List Token:
 Referenced Token:
 : A cryptographically secured data structure which contains a reference to a Status List or Status List Token. It is RECOMMENDED to use JSON {{RFC8259}} or CBOR {{RFC8949}} for representation of the token and secure it using JSON Object Signing as defined in {{RFC7515}} or CBOR Object Signing and Encryption as defined in {{RFC9052}}. The information from the contained Status List may give a Relying Party additional information about up-to-date status of the Referenced Token.
 
+base64url:
+: Denotes the URL-safe base64 encoding without padding as defined in Section 2 of [RFC7515] as "Base64url Encoding".
```

Comment:
Does this render correctly?

Reply:
Same as the rest in the Terminology section: https://drafts.oauth.net/draft-ietf-oauth-status-list/c2bo/44-clarify-base64-encoding/draft-ietf-oauth-status-list.html#name-terminology

Reply:
I meant square brackets instead of double curly brackets

Reply:
ohh - I didn't even realize that :D, changed it